### ISSUE TITLE
Feature request: confirmation for large jobs

### ISSUE THREAD

**Issue by Efreak** [daveschumaker/artbot-for-stable-diffusion#85]
TLDR: if the number of kudos to be consumed in a job is too high, please request confirmation

This is also mentioned at the end of #84. 

Multiple times on mobile I've caught myself accidentally tapping the number of images slider. Today while at work, I accidentally submitted a request with ~15 models selected asking for *187* images from each model instead of just 1 (all with the same seed at that). While I do believe advanced options should be available[^1], 

- the number of total images is too high (10?). Should note that the number is displayed below the submit button, and have an option to never show again[^1]
- the number of kudos the request will use is more than a certain percentage of what you have available.
- when the absolute number of kudos used is high (5k?).
  - example: I'm going for all models to create a comparison, and forget that my canvas and step count are set too high from a previous job *because I'm dumb and don't use the reset button when I should*
  - Doesn't have to, but could possibly identify which settings are being used beyond "normal" values, to make it easier to fix, ie number of models requested if higher than 5(?), all samplers toggle if set, size of canvas if # of mp > 0.5(?), step count if higher than 50(?)

These *could* all be configurable numbers to prevent annoying power users but still be helpful to them[^1]. If not, it *should* be possible to disable them settings, as the first two suggestions here are mainly for people rating images for kudos, and people finding {multiprompts} online and thinking they're asking for one image (the number of images text is only useful if you think to look at it) 

[1] I've only set the slider past 10 a couple times, I think, but I can absolutely see myself getting frustrated at arrangement of results and intentionally setting it to 50+ to try and get what I want. several times Ive made use of the ability to submit jobs that require 10k+ kudos for things like showcases or comparisons
